diff --git a/RecoEgamma/EgammaTools/src/EnergyScaleCorrection.cc b/RecoEgamma/EgammaTools/src/EnergyScaleCorrection.cc
index f2e9161057b..935aded506a 100644
--- a/RecoEgamma/EgammaTools/src/EnergyScaleCorrection.cc
+++ b/RecoEgamma/EgammaTools/src/EnergyScaleCorrection.cc
@@ -69,13 +69,14 @@ float EnergyScaleCorrection::smearingSigma(
   const SmearCorrection* smearCorr = getSmearCorr(runnr, et, eta, r9, gainSeed);
 
   if (smearCorr != nullptr)
-    return smearCorr->sigma(nrSigmaRho, nrSigmaPhi);
+    return smearCorr->sigma(et, nrSigmaRho, nrSigmaPhi);
   else
     return kDefaultSmearVal_;
 }
 
 const EnergyScaleCorrection::ScaleCorrection* EnergyScaleCorrection::getScaleCorr(
     unsigned int runnr, double et, double eta, double r9, unsigned int gainSeed) const {
+  eta = fabs(eta);
   // buld the category based on the values of the object
   CorrectionCategory category(runnr, et, eta, r9, gainSeed);
   auto result = scales_.find(category);
@@ -97,6 +98,7 @@ const EnergyScaleCorrection::ScaleCorrection* EnergyScaleCorrection::getScaleCor
 
 const EnergyScaleCorrection::SmearCorrection* EnergyScaleCorrection::getSmearCorr(
     unsigned int runnr, double et, double eta, double r9, unsigned int gainSeed) const {
+  eta = fabs(eta);
   // buld the category based on the values of the object
   CorrectionCategory category(runnr, et, eta, r9, gainSeed);
   auto result = smearings_.find(category);
